### PR TITLE
feat: local relayer

### PR DESCRIPTION
**Pull request by ratankaliani**
Integrate local relaying into `BlobstreamX`.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -65,7 +68,8 @@ subtle-encoding = "0.5.1"
 tendermint = "0.33.0"
 tendermint-proto = "0.33.0"
 tokio = { version = "1.29.1", features = ["full"] }
-tendermintx = { git = "https://github.com/succinctlabs/tendermintx.git" }
+tendermintx = { git = "https://github.com/succinctlabs/tendermintx.git", branch = "ratan/verify-chain-id" }
```
Comment (puma314) [inline Cargo.toml:71]: note to change back after these get merged

File: bin/blobstreamx.rs [hunk 2]
```diff
@@ -48,10 +54,31 @@ impl BlobstreamXOperator {
 
         let succinct_rpc_url = env::var("SUCCINCT_RPC_URL").expect("SUCCINCT_RPC_URL must be set");
         let succinct_api_key = env::var("SUCCINCT_API_KEY").expect("SUCCINCT_API_KEY must be set");
-        let client = SuccinctClient::new(succinct_rpc_url, succinct_api_key);
+
+        let private_key: Option<String>;
```
Comment (puma314) [inline bin/blobstreamx.rs:58]: Oh yeah keeping these around as `Option` is why I thought reading from the .env within the actual method would be cleaner lol

File: bin/local_relay.rs
Comment (puma314) [inline bin/local_relay.rs:1]: do we still need this file?

Comment (ratankaliani) [reply bin/local_relay.rs:1]: No we don't, I'll go ahead and remove!

File: bin/blobstreamx.rs (hunk 2 above)
Comment (ratankaliani) [reply bin/blobstreamx.rs:58]: Yeah, but I think this is more correct if people specify their `PRIVATE_KEY` differently.

**Pull request merged**